docs/_advanced/receiver.md

```diff
@@ -6,6 +6,9 @@ order: 9
 ---
 
 <div class="section-content">
+
+#### Writing a custom receiver
+
 A receiver is responsible for handling and parsing any incoming requests from Slack then sending it to the app, so that the app can add context and pass the request to your listeners. Receivers must conform to the Receiver interface:
```

Comment:
I know this isn't exactly part of your PR, but perhaps we can link `Receiver interface` to https://github.com/slackapi/bolt-js/blob/main/src/types/receiver.ts#L27-L31?

```suggestion
A receiver is responsible for handling and parsing any incoming requests from Slack then sending it to the app, so that the app can add context and pass the request to your listeners. Receivers must conform to the [Receiver interface](https://github.com/slackapi/bolt-js/blob/main/src/types/receiver.ts#L27-L31):
```

Reply:
@filmaj We should avoid pointing code lines in the main branch as the code can be changed in the future. How about using a release tag instead? https://github.com/slackapi/bolt-js/blob/%40slack/bolt%403.13.1/src/types/receiver.ts#L27-L31